Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:11
```

Comment:
Optional: Ideally we want to have a multi-stage docker here with separate JDK for building the fat-jar. Copy the fat-jar into a JRE image for runtime.